### PR TITLE
WIP

### PR DESCRIPTION
**Pull request by firehist**
I added few routes (cf README or `/` route).
We need to list what we wanna do, cause the NodeJS API update allows us to catch pokemon ;)

### CODE REVIEW

File: src/PokeUsers.class.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const PokeUser = require('./PokeUser.class')
```
Comment (heiso) [inline src/PokeUsers.class.js]: Prefer creating a collection class that can embed any class. And add a static on PokeUser class that use this collection to create many users.

File: src/main.js [hunk 2]
```diff
@@ -24,27 +46,44 @@ app.listen(3000, () => {
...
-    .map(val => `${val.route.stack[0].method.toUpperCase()} ${val.route.path}`)
+  res.status(200).json(
+    app._router.stack
+      .filter(val => val.route)
+      .map(val => `${val.route.stack[0].method.toUpperCase()} ${val.route.path}`)
+  )
+})
 
-  res.json(200, routes)
+app.get('/users', (req, res) => {
```
Comment (heiso) [inline src/main.js:57]: '/user'

File: src/main.js [hunk 3]
```diff
@@ -24,27 +46,44 @@ app.listen(3000, () => {
...
 })
 
 app.get('/getHeartbeat/:coords', (req, res) => {
-  var latitude = parseFloat(req.params.coords.split(',')[0], 10)
-  var longitude = parseFloat(req.params.coords.split(',')[1], 10)
-  user
-    .setLocation(latitude, longitude)
+  let {lat, lon} = Utils.getCoordFromQueryString(req.params.coords)
+
+  console.log(lat, lon)
```
Comment (heiso) [inline src/main.js]: console.log

**Comment by heiso** [comment]
LGTM

File: src/PokeUsers.class.js (hunk 1 above)
Comment (firehist) [inline src/PokeUsers.class.js]: Didn't get what you want ;)
I created a `PokeUsers` to store in-memory all users (by UUID).
express-session create on-th-fly a user if not found.

File: src/main.js (hunk 3 above)
Comment (firehist) [inline src/main.js]: lovely ;)

File: src/main.js (hunk 2 above)
Comment (firehist) [inline src/main.js:57]: WTF?
It returns the list of all users in-memory.
It allows us to use different accounts for different user!

**Pull request closed without merging**